rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -533,7 +535,31 @@ def test_exec(config):
             if final_op != -1:
                 test_info.failed_status("test failed")
                 sys.exit(1)
-
+        if config.etag_verification is True:
+            log.info(f"Verification of eTag is started!!! ")
+            object_ptr = s3lib.resource_op(
+                {
+                    "obj": bucket,
+                    "resource": "Object",
+                    "args": [s3_object_name],
+                }
+            )
+            object_info = object_ptr.get()
+            eTag_aws = object_info["ETag"].split('"')[1]
+            log.info(f"etag from aws is :{eTag_aws}")
+            cmd = f"radosgw-admin bucket list --bucket {bucket.name}"
+            out = utils.exec_shell_cmd(cmd)
+            data = json.loads(out)
+            for object in data:
+                if str(s3_object_name) == str(object["name"]):
+                    eTag_radosgw = object["meta"]["etag"]
+                    log.info(f"etag from radosgw is :{eTag_radosgw}")
+                    if str(eTag_aws) == str(eTag_radosgw):
+                        log.info(f"eTag matched!!")
+                    else:
+                        raise AssertionError(
+                            f"mismatch found in the eTAG from aws and radosgw"
+                        )
```

Comment:
The code will check only last object's etag info if we give more than 1 value  in `object_count`

Reply:
done